src/imports/Ubuntu/Contacts/ContactDelegate.qml

```diff
@@ -27,7 +27,7 @@ ListItemWithActions {
     property bool isCurrentItem: false
     property string defaultAvatarUrl: ""
     property bool flicking: false
-    property int contactIndex: index
+    property int contactIndex: -1
```

Comment:
Why do you set this to -1?

Reply:
To fix a `TypeError` and because it's an `int` while the value is checked against `>= 0` elsewhere (since list index can't be negative nor undefined).

Reply:
Where is it set? Is it still the correct value now?

Reply:
OK. Poked more and tried to set it to a value elsewhere, but seemed to just move the error, so just reverted this bit.